component/service/src/main/java/org/exoplatform/social/rest/impl/space/SpaceRestResourcesV1.java

```diff
@@ -253,7 +253,48 @@ public Response createSpace(@Context UriInfo uriInfo,
 
     return EntityBuilder.getResponse(EntityBuilder.buildEntityFromSpace(space, authenticatedUser, uriInfo.getPath(), expand), uriInfo, RestUtils.getJsonMediaType(), Response.Status.OK);
   }
-  
+
+  /**
+   * {@inheritDoc}
+   */
+  @GET
+  @Produces(MediaType.APPLICATION_JSON)
+  @Path("{spaceId}/checkExternals")
+  @RolesAllowed("users")
+  @ApiOperation(value = "Checks if a specific a space contains an external members",
+          httpMethod = "GET",
+          response = Response.class,
+          notes = "This returns the space if it contains external members")
+  @ApiResponses(value = {
+          @ApiResponse(code = 200, message = "Request fulfilled"),
+          @ApiResponse(code = 500, message = "Internal server error"),
+          @ApiResponse(code = 400, message = "Invalid query input")})
+  public Response isSpaceContainsExternals(@Context UriInfo uriInfo,
+                                           @Context Request request,
+                                           @ApiParam(value = "Space Id", required = true) @PathParam("spaceId") String spaceId) {
+    String authenticatedUser = ConversationState.getCurrent().getIdentity().getUserId();
+
+    if (spaceId == null) {
+      return Response.status(Status.BAD_REQUEST).entity("space Id is mandatory").build();
+    }
+
+    Space space = spaceService.getSpaceById(spaceId);
+    if (space == null) {
+      return Response.status(Status.NOT_FOUND).entity("space not found").build();
+    }
+    boolean hasExternals;
+    try {
+      hasExternals = spaceService.isSpaceContainsExternals(Long.valueOf(spaceId));
+    } catch (Exception e) {
+      LOG.error("Error while checking external members on space {}", space.getGroupId());
+      return Response.status(Status.INTERNAL_SERVER_ERROR).entity("server internal error occurred").build();
+    }
+    if (hasExternals) {
```

Comment:
Should we only answer true or false ?

Reply:
I thought it's better to have the space info in the rest response

Reply:
do we need it ? If not, we should only answer true or false

Reply:
I need the space display name

Reply:
When you call 
this.$spaceService.checkExternals(space.spaceId).then(spaceObj => {
in the vue file, you already have the space object. Can you take the space display name here ? instead of returning the complete space object ?

Reply:
OK

Reply:
done